napari/layers/_shapes_layer/model.py

```diff
@@ -0,0 +1,1521 @@
+        return self._edge_color
+
+    @edge_color.setter
+    def edge_color(self, edge_color):
+        self._edge_color = edge_color
+        if self._apply_all:
+            index = list(range(len(self.data.shapes)))
+        else:
+            index = self.selected_shapes
+        for i in index:
+            self.data.update_edge_color(i, edge_color)
+        self.refresh()
+        self.events.edge_color()
+
+    @property
+    def face_color(self):
+        """str: color of faces
+        """
+        return self._face_color
+
+    @face_color.setter
+    def face_color(self, face_color):
+        self._face_color = face_color
+        if self._apply_all:
+            index = list(range(len(self.data.shapes)))
+        else:
+            index = self.selected_shapes
+        for i in index:
+            self.data.update_face_color(i, face_color)
+        self.refresh()
+        self.events.face_color()
+
+    @property
+    def opacity(self):
+        """float: Opacity value between 0.0 and 1.0.
+        """
+        return self._opacity
+
+    @opacity.setter
+    def opacity(self, opacity):
+        if not 0.0 <= opacity <= 1.0:
+            raise ValueError('opacity must be between 0.0 and 1.0; '
+                             f'got {opacity}')
+
+        self._opacity = opacity
+        if self._apply_all:
+            index = list(range(len(self.data.shapes)))
+        else:
+            index = self.selected_shapes
+        for i in index:
+            self.data.update_opacity(i, opacity)
+        self.refresh()
+        self.events.opacity()
+
+    @property
+    def apply_all(self):
+        """bool: whether to apply a manipulation to all shapes or just the
+        selected ones
+        """
+        return self._apply_all
+
+    @apply_all.setter
+    def apply_all(self, apply_all):
+        self._apply_all = apply_all
+        self.events.apply_all()
+
+    @property
+    def selected_shapes(self):
+        """list: list of currently selected shapes
+        """
+        return self._selected_shapes
+
+    @selected_shapes.setter
+    def selected_shapes(self, selected_shapes):
+        self._selected_shapes = selected_shapes
+        self._selected_box = self.bounding_box(selected_shapes)
+
+    @property
+    def mode(self):
+        """str: Interactive mode
+        """
+        return self._mode
+
+    @mode.setter
+    def mode(self, mode):
+        if mode == self.mode:
+            return
+        old_mode = self.mode
+        if mode == 'select':
+            self.cursor = 'pointing'
+            self.interactive = False
+            self.help = ('hold <space> to pan/zoom, '
+                         'press <delete> to remove selected')
+            self.status = mode
+            self._mode = mode
+        elif mode == 'direct':
+            self.cursor = 'pointing'
+            self.interactive = False
+            self.help = ('hold <space> to pan/zoom, '
+                         'press <delete> to remove selected')
```

Comment:
I presume this means "backspace" in Windows/Linux?

Reply:
I think so - should I use `<backspace>` instead?

Reply:
Well, "delete" is the right label in macOS (even though the ASCII code inserted is backspace). I think you should make a variable:

```python
backspace = 'delete' if sys.platform == 'darwin' else 'backspace'
```

Then:

```suggestion
                         f'press <{backspace}> to remove selected')
```

Reply:
sounds good, I've fixed